c/meterpreter/source/extensions/stdapi/server/net/socket/tcp_server.c

```diff
@@ -19,6 +19,52 @@
 extern IN6_ADDR in6addr_any;
 #endif
 
+static int get_ai_family(const char* address) {
+	struct addrinfo* resolved_host = NULL;
+	struct addrinfo hints = {
+		.ai_family = AF_UNSPEC,
+		.ai_flags = AI_NUMERICHOST
+	};
+
+	int ai_family = AF_UNSPEC;
+	if (getaddrinfo(address, NULL, &hints, &resolved_host) == 0) {
+		ai_family = resolved_host->ai_family;
+		freeaddrinfo(resolved_host);
+	}
+	return ai_family;
+}
+
+// The inet_pton function is not available prior to Windows 8.1 so
+// use getaddrinfo.

```

Comment:
I don't know if this is actually as valid comment. According to https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-inet_pton it is supported on Windows Vista and later, and `inet_pton` is simply an alias for the ASCII version of this function, aka `InetPtonA`, as noted in the line `The ANSI version of this function is inet_pton as defined in RFC 2553. For more information, see RFC 2553 available at the [IETF website](http://tools.ietf.org/html/rfc2553).`

What you may have been looking at is the note at the bottom of the page r.e Windows Store Apps, which are different apps than the ones we are using, and their support for `InetPtonW` on Windows 8.1 and Windows Server 2012 R2 and later. Since we aren't a Windows Store app I don't think this applies to us and this seems to be more of a side note specific to Windows Store apps.